addons/website_crm/controllers/main.py

```diff
@@ -77,7 +78,6 @@ def dict_to_str(title, dictvar):
             return request.website.render(kwargs.get("view_from", "website.contactus"), values)
 
         try:
-            values['medium_id'] = request.registry['ir.model.data'].get_object_reference(request.cr, SUPERUSER_ID, 'crm', 'crm_tracking_medium_website')[1]
```

Comment:
I think

 - replace crm_tracking_medium_website by crm_medium_website that exists
 - change get_object_refernece by xmlid_to_res_id, that returns False if the object does not exist anymore
 - remove the try/catch, values will be False / False if nothing exist

This should solve several issues...